compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/IterableAnalyzer.java

```diff
@@ -155,20 +156,20 @@ private void handleLambdaBasedIterableOperation(IterableContext context, Operati
         final List<BType> givenArgTypes = calculatedGivenInputArgs(operation);
         final List<BType> givenRetTypes = calculatedGivenOutputArgs(operation);
 
-        final List<BType> expectedArgTypes = calculateExpectedInputArgs(operation);
-        final List<BType> expectedRetTypes = calculateExpectedOutputArgs(operation, givenRetTypes);
+        final List<BType> actualArgTypes = calculateExpectedInputArgs(operation);
```

Comment:
the function name and the variable name doesn't seems to match.
Line 156 calculates the givenArgType (actuals), so shouldn't this be the `expectedArgTypes`?